Introduction to Kotlin Multiplatform/06_jvm_render.md

```diff
@@ -1,33 +1,39 @@
-# Rendering in JavaScript
+# Common Rendering and JVM
 
 It is always good to question ourselves - do we really need the
 JVM backend for the project or not. What if we would be able
-to do the [Mandelbrot Set](https://en.wikipedia.org/wiki/Mandelbrot_set)
+to run the [Mandelbrot Set](https://en.wikipedia.org/wiki/Mandelbrot_set)
 rendering on the client? 
 
 Let's implement the client-side rendering and see how it goes.
 
 ## Sharing Code
 
 We've just added the very first Kotlin file into the `commonMain`
-source set. Now let's move all files from the `jvmMain` folder
-into the `commonMain` folder to share the code with JavaScript.
+source set. Let's move all the files from the `jvm` source set
+into the `common` source set to share the code with JavaScript.
 
-Wait a moment. It will not work this way because we use JVM APIs
-to render images. On the client-side we should be using
+Wait a moment. It will not work directly this way because we use JVM APIs
```

Comment:
Wait just a second. It won't work this way, because we are using JVM APIs
to render images in our code. We need to pick another API to use in
the browser Kotlin/JS part.
We will instead use the
[HTML Canvas](https://www.w3schools.com/html/html5_canvas.asp)
to render our images in JavaScript.